protos/flyteidl/admin/matchable_resource.proto

```diff
@@ -64,3 +64,23 @@ message MatchingAttributes {
     ExecutionClusterLabel execution_cluster_label = 4;
   }
 }
+
+message MatchableResourceConfiguration {
+    MatchingAttributes attributes = 1;
+
+    string domain = 2;
+
+    string project = 3;
+
+    string workflow = 4;
+}
+
+message ListMatchableResourcesRequest {
+    string domain = 1;
```

Comment:
Why only domain?

Reply:
we scope everything in the system by domain? but i can remove if you think it's not a great requirement!